travispy/entities/hook.py

```diff
@@ -30,3 +31,29 @@ class Hook(Entity):
         'private',
         'admin',
     ]
+
+    def _set_hook(self, flag):
```

Comment:
One possible problem with taking out `Hook`: I couldn't figure out how to get access to the complete list of active and inactive repos without using the `Hook` entity.  When I used `t.repos()`, I think I was getting only active ones.  Am I missing something?